app/views/relatives/_show.html.slim

```diff
@@ -0,0 +1,9 @@
+td= t('relatives')
+td
+  - if relative.relatives.any?
+    dl.dl-horizontal
+      - relative.relatives.each do |rel|
+        dt #{rel.first_name} #{rel.last_name}
```

Comment:
Please make a `full_name` method in the model and just output `dt= rel.full_name`